net-analyzer/gvm-tools/gvm-tools-1.4.1.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6} )
+inherit distutils-r1
+
+DESCRIPTION="A remote security scanner for Linux (OpenVAS-cli)"
+HOMEPAGE="http://www.openvas.org/"
+SRC_URI="https://github.com/greenbone/gvm-tools/archive/v1.4.1.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="
```

Comment:
Are these really needed when building, or just when running (ie should all of the items in this list be moved to RDEPEND)

Reply:
Fixed gvm-tools.
also ospd is fixed.same situation.

I am not sure still we need **dev-python/setuptools** here ?
```
RDEPEND="
	dev-python/defusedxml[${PYTHON_USEDEP}]
        dev-python/lxml[${PYTHON_USEDEP}]
        dev-python/paramiko[${PYTHON_USEDEP}]
        dev-python/pythondialog:0[${PYTHON_USEDEP}]
        dev-python/setuptools[${PYTHON_USEDEP}]
        !net-analyzer/openvas-cli"

DEPEND="
        ${RDEPEND}"
```

Reply:
Yes, leave it for RDEPEND. distutils-r1 puts it in BDEPEND for you, but not for RDEPEND.
(I have a feeling it's needed in RDEPEND to check runtime dependencies/optionalities)